torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1745,3 +1745,76 @@ def rref_tensor_is_owner(rref_var):
 
         res = rref_tensor_is_owner(rref_var)
         self.assertEqual(res, False)
+
+    @dist_init
+    def test_rpc_async_op(self):
```

Comment:
Can we add test cases for invalid cases like:

1. args and/or kwargs are invalid for the function specified.
2. destination function is not a torchscript function.
3. destination function throws an exception and appropriate exception is received when calling fut.wait.
4. destination function does not exist.

Reply:
Added these invalid test cases

- lesser than necessary args
- more than necessary args
- unexpected kwargs
- destination function is not a torchscript function.
- destination function throws an exception and appropriate exception is received when calling fut.wait.
- destination function does not exist.